pumpkin/build.rs

```diff
@@ -2,7 +2,8 @@ use git_version::git_version;
 use std::env;
 
 fn main() {
-    if cfg!(target_os = "windows") {
+    #[cfg(target_os = "windows")]
```

Comment:
Also add `#[cfg(not(debug_assertions))]` because we don't need icons on debug build, and we have the same issue with recompiling on widows if we don't add it.

Reply:
good catch!